uaclient/util.py

```diff
@@ -463,6 +477,8 @@ def subp(
     """Run a command and return a tuple of decoded stdout, stderr.
 
      @param subp: A list of arguments to feed to subprocess.Popen
+     @param data: A str or bytes to pass as stdin to the command.
+     @param capture: Boolean set True to log the command and response.
```

Comment:
Let's maintain argument order in the docstring (so move this down one line).

Reply:
dropped on revert of apt-key cmd calls. point taken.